pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -445,42 +421,47 @@ private PinotResourceManagerResponse uploadSegment(File indexDir, SegmentMetadat
         expectedCrc = Long.parseLong(expectedCrcStr);
       } catch (NumberFormatException e) {
         throw new ControllerApplicationException(LOGGER,
-            "Caught exception while parsing IF-MATCH CRC: " + expectedCrcStr, Response.Status.PRECONDITION_FAILED, e);
+            "Caught exception for segment: " + segmentName + " of table: " + offlineTableName
```

Comment:
do we also want to include e?

Reply:
Not necessary because from the message we already know the exception type